packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -135,7 +385,28 @@ export class WcSdk {
         return await WcSdk.testInvoke(this.wcClient, this.session, this.chainId, request)
     }
 
-    async multiInvoke(request: ContractInvocationMulti) {
+    /**
+     * Sends a `multiInvoke` request to the Wallet, it will concatenate all invocations and send to the blockchain all at once.

```

Comment:
```suggestion
     * Sends a `multiInvoke` request to the Wallet. It will concatenate all invocations and send them to the blockchain all at once.

```